freqtrade/configuration/configuration.py

```diff
@@ -201,12 +198,11 @@ def _load_datadir_config(self, config: Dict[str, Any]) -> None:
             config.update({'datadir': create_datadir(config, None)})
         logger.info('Using data directory: %s ...', config.get('datadir'))
 
-    def _load_optimize_config(self, config: Dict[str, Any]) -> Dict[str, Any]:
+    def _load_optimize_config(self, config: Dict[str, Any]) -> None:
```

Comment:
I don't like the naming of `_load_*` (it's not really correct)

we have 8 functions starting with `_load` - only one of which actually loads something (`_load_config_files()`), all others simply copy arguments from args to config.

We may be able to consolidate these into one (since the syntax for these is now a lot shorter).

Maybe not for this PR - just something i noticed now.

Reply:
Heh, I already renamed them to `_process_*_options()` (like `_process_optimize_options()`)... I split wide changes to the smaller ones so that the changes remain observable, so I planned to commit it in the next PR along with further refactoring, but particularly this renaming can be pushed in the PR, I guess...

Reply:
This one is done and committed. If you agree to postpone your previous proposal, I assume this is ready for final review...